src/Hedgehog.Experimental.CSharp.Tests/GenTests.cs

```diff
@@ -52,4 +53,18 @@ public void ShouldAddElementToList()
 
     prop.Check();
   }
+
+  [Fact]
+  public void ShouldSupportIEnumerable() =>
+    GenX.auto<IEnumerable<int>>()
+      .Sample(1, 5)
+      .Should()
+      .AllSatisfy(x => x.Should().NotBeNull());
+
+  [Fact]
+  public void ShouldSupportList() =>
+    GenX.auto<List<int>>()
```

Comment:
I think the equivalent type in F# is `ResizeArray<int>`.

Can you check if the equivalent test fails for F#?

If it does, can you add it to your first commit?

Reply:
It does not fail, I was just checking that `Enumerable` pattern doesn't pick up anything but `IEnumerable`.

Reply:
Ok.  Then please split your first commit into two: the first commit containing this passing test and the second commit containing your two failing tests.

Reply:
I'll rather remove the tests that I added, these splitting/merging are getting involved.

Reply:
Thanks for doing that (and for your patience :) ).